.size-limit.js

```diff
@@ -28,13 +28,13 @@ module.exports = [
     name: 'The size of the @material-ui/core modules',
     webpack: true,
     path: 'packages/material-ui/build/index.js',
-    limit: '91.3 KB',
+    limit: '91.2 KB',
   },
   {
     name: 'The size of the @material-ui/styles modules',
     webpack: true,
     path: 'packages/material-ui-styles/build/index.js',
-    limit: '16.3 KB',
+    limit: '16.4 KB',
```

Comment:
For some reason `@material-ui/utils` appears in the styles bundle. Tree shaking is working fine but the dependency is not removed in a production bundle. It doesn't appear in the core bundle though.

Reply:
It does not appear in the esm bundle. I would keep this for now and revisit what bundles we monitor for size later.